static/app/views/discover/table/tableView.tsx

```diff
@@ -190,15 +190,14 @@ function TableView(props: TableViewProps) {
 
       let target;
 
-      if (dataRow.trace === null) {
+      if (dataRow.trace == null) {
```

Comment:
This is misleading and can be really hard to catch when looking for bugs as it is actually checking datarow.trace === null || datarow.trace === undefined. You should either use !datarow.trace or do an explicit check for both value types